docs/junipernetworks.junos.junos_security_policies.rst

```diff
@@ -0,0 +1,1551 @@
+.. _junipernetworks.junos.junos_security_policies_module:
```

Comment:
This need to be updated for junos_security_policies , you can use Collection prep engine for this https://github.com/ansible-network/collection_prep